easybuild/easyconfigs/g/GCC/GCC-5.2.0-binutils-2.25.eb

```diff
@@ -0,0 +1,55 @@
+name = 'GCC'
+version = '5.2.0'
+
+binutilsver = '2.25'
+versionsuffix = '-binutils-%s' % binutilsver
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.3'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/%(namelower)s/%(namelower)s-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+]
+
+sources = [
+    SOURCELOWER_TAR_BZ2,
+    'gmp-6.0.0a.tar.bz2',
+    'mpfr-%s.tar.gz' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.14.tar.bz2',
+]
+
+patches = [('mpfr-%s-allpatches-20150717.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
+
+builddependencies = [
+    ('M4', '1.4.17'),
+    ('binutils', binutilsver),
+]
+
+checksums = [
+    'a51bcfeb3da7dd4c623e27207ed43467',     # gcc-5.2.0.tar.bz2
+    'b7ff2d88cae7f8085bd5006096eed470',     # gmp-6.0.0a.tar.bz2
+    '7b650781f0a7c4a62e9bc8bdaaa0018b',     # mpfr-3.1.l.tar.gz
+    'd6a1d5f8ddea3abd2cc3e98f58352d26',     # mpc-1.0.3.tar.gz
+    'acd347243fca5609e3df37dba47fd0bb',     # isl-0.14.tar.bz2
+    'e502185ebb22b41c528f183bb22a7569',     # mpfr-3.1.3-allpatches-20150717.patch
+]
+
+languages = ['c', 'c++', 'fortran']
+
+withisl = False
```

Comment:
Unfortunately, keeping that on `True` makes the build fail in configuration of stage 3 (btw: is that a typo in the logging output: `Stage 2 completed [...] continue with stage 2`?):
```
== 2015-09-26 15:37:42,306 main.EB_GCC INFO Stage 2 of 3-staged build completed, continuing with stage 2 (with CLooG and/or PPL, ISL support enabled)...
== 2015-09-26 15:37:42,812 main ERROR cmd " ../configure  --enable-languages=c,c++,fortran --enable-lto --enable-checking=release  --disable-multilib --enable-shared=yes --enable-static=yes  --enable-threads=posix  --enable-gold=default --enable-plugins  --enable-ld --wi
th-plugin-ld=ld.gold  --prefix=/opt/eb/software/GCC/5.2.0-binutils-2.25 --with-local-prefix=/opt/eb/software/GCC/5.2.0-binutils-2.25 --enable-bootstrap --with-isl=/data/opt/eb/build/GCC/5.2.0/dummy--binutils-2.25/gcc-5.2.0/stage2_stuff " exited with exitcode 1 and output
:
configure: loading site script /usr/share/site/x86_64-unknown-linux-gnu
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether ln works... yes
checking whether ln -s works... yes
checking for a sed that does not truncate output... /usr/bin/sed
checking for gawk... gawk
checking for libatomic support... yes
checking for libcilkrts support... yes
checking for libitm support... yes
checking for libsanitizer support... yes
checking for libvtv support... yes
checking for gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking whether g++ accepts -static-libstdc++ -static-libgcc... yes
checking for gnatbind... no
checking for gnatmake... no
checking whether compiler driver understands Ada... no
checking how to compare bootstrapped objects... cmp --ignore-initial=16 $$f1 $$f2
checking for objdir... .libs
checking for compatible ISL... no
configure: error: Unable to find a usable ISL.  See config.log for details.
```